src/shared/helpers.h

```diff
@@ -30,20 +30,9 @@
 
 #define UNCONST(a)		((void *)(unsigned long)(const void *)(a))
```

Comment:
oh wow i only noticed this now.  this is horribly broken.  my desktop is broken atm so i can't fix.  this needs to be `uintptr_t`, not `unsigned long`.

Reply:
good spot, done in https://github.com/OpenRC/openrc/pull/624